apcd-cms/src/apps/admin_regis_table/static/admin_regis_table/css/table.css

```diff
@@ -10,10 +10,7 @@
     letter-spacing: 0px;
     color: #484848;
     opacity: 1;
-    padding-top: var(--cell-padding);
-    padding-bottom: var(--cell-padding);
-    padding-left: 10px;
-    padding-right: 40px;
+    padding: var(--cell-padding);
```

Comment:
**Explanation.** Tiny tweak for consistency. Also, the smaller padding delays wrapping of text.
<sub>Exact table styles are pending [FP-1878](https://jira.tacc.utexas.edu/browse/FP-1878).</sub>